Resources/Private/Partials/Filter/Inline.html

```
@@ -0,0 +1,20 @@
+<label class="form-label" for="search_field">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)